README.md

```diff
@@ -1,46 +1,197 @@
-# Requirements
+# Overview 
 
-- Golem (branch `glambda0.3^3c8f9d425debb0e72017e427f7dc8edbca099185`)
+Golemrpc is a python package allowing communication with a (remote) Golem node. Connection handling, golem task state handling, resources upload and results retrieval are handled automatically for the user. This package has been created mainly for RASPA2 use case, but it also works with basic blender rendering tasks (see examples). 
 
-# Quickstart
+RASPA2 tasks are given to Golem in form of Python functions and `args` dictionary, e.g.:
 
-## Install with virtualenv
+```python
+def raspa_task(args):
+    '''Task to compute provider side.
+    It is possible to import RASPA2 package on provider side because
+    this package is preinstalled in a docker environment we are about
+    to use (dockerhub golemfactory/glambda:1.3).
+    '''
+    import RASPA2
+    import pybel
 
+    mol = pybel.readstring('cif', args['mol'])
 
-```sh
-git clone git@github.com:golemfactory/golemrpc.git &&\
-cd golemrpc &&\
-python3 -m venv venv &&\
-source venv/bin/activate &&\
-pip install --upgrade pip  &&\
-pip install -e .
+    return RASPA2.get_helium_void_fraction(mol)
 ```
 
-## Run
+It is called RASPA2, however generic Python functions are equally acceptable. 
```

Comment:
What is called RASPA2?

Reply:
Our "use-case". What I wanted to state here is the fact that you can run any python function here.